src/lib.rs

```diff
@@ -142,17 +145,24 @@ pub mod doc_test {
     doctest!("../README.md", readme_md);
 }
 
-const EXPECT_INIT: &str = "PyO3 Asyncio has not been initialized";
-
 static ASYNCIO: OnceCell<PyObject> = OnceCell::new();
 static ENSURE_FUTURE: OnceCell<PyObject> = OnceCell::new();
-static EVENT_LOOP: OnceCell<PyObject> = OnceCell::new();
+
+const EXPECT_INIT: &str = "PyO3 Asyncio has not been initialized";
+static CACHED_EVENT_LOOP: OnceCell<PyObject> = OnceCell::new();
```

Comment:
shouldn't this be thread-local or the likes?

Reply:
The `CACHED_EVENT_LOOP` is to support the deprecated `0.13` API. It's ignored entirely in the new functions.